src/parseTools.js

```diff
@@ -865,6 +865,13 @@ function hasExportedSymbol(sym) {
   return WASM_EXPORTS.has(sym);
 }
 
+function maybeExport(sym) {
```

Comment:
How is this related to the function here with the same name?

https://github.com/emscripten-core/emscripten/blob/3e780ad0f69e97f6883616d108e1c99c23107100/src/modules.js#L347

Reply:
I renamed the new function `receivedSymbol` for now, and added a comment.

We could potentially clean this up as a followup by introducing the concept of a delayed export (i.e. and export that cannot be set until after instantiation.    But for now I think this change is big enough.